google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/DefaultChangeStreamRecordAdapter.java

```diff
@@ -100,10 +102,10 @@ public ChangeStreamRecord onCloseStream(ReadChangeStreamResponse.CloseStream clo
 
     /** {@inheritDoc} */
     @Override
-    public void startUserMutation(
+    public void startUserMutationInstant(
```

Comment:
I know this is marked with `@InternalApi`. Let's double check that this is fine (hopefully no BigTable users are using internalApi marked apis)

Reply:
@igorbernstein2 is it ok to change the signature of this method directly since it's marked as `@InternalApi`?